src/app/(app)/product-list.tsx

```diff
@@ -20,7 +20,9 @@ const FiltersButton = ({ products }: { products: Product[] }) => {
     return;
   }
   return (
-    <View className="absolute -bottom-8 flex w-auto items-center justify-center self-center pb-20">
+    <View
+      className={`absolute bottom-24 flex w-auto items-center justify-center self-center pb-20`}
```

Comment:
you are not using the string interpolation

```suggestion
      className="absolute bottom-24 flex w-auto items-center justify-center self-center pb-20"
```